pkg/events/client.go

```diff
@@ -89,7 +89,7 @@ type Event struct {
 // ListEventsOpts contains the options to request provisioned Events.
 type ListEventsOpts struct {
 	// Filter to only return Events of particular types.
-	Events []string `url:"events,omitempty"`
+	Events []string `url:"events"`
```

Comment:
@PaulAsjes While this change is breaking in the API (mitigated by feature flag for affected customers), I don't consider this is a breaking change in the Go SDK, so a minor bump seems appropriate. Wanted to double check to see if you had an opinion though.

Reply:
Agreed, minor bump seems fine!